.github/workflows/scripts/qemu-4-build.sh

```diff
@@ -56,7 +56,8 @@ function linux() {
     --prefix=/usr \
     --enable-pyzfs \
     --enable-debug \
-    --enable-debuginfo
+    --enable-debuginfo \
+    --sysconfdir=/etc
```

Comment:
Why is this needed on Arch but not elsewhere?

Reply:
Looks like the configs get installed to /usr/etc/zfs instead of /etc/zfs.

Reply:
Untested, but I would suggest:
```diff
diff --git a/.github/workflows/scripts/qemu-4-build.sh b/.github/workflows/scripts/qemu-4-build.sh
index 955f605f5..a4335fb63 100755
--- a/.github/workflows/scripts/qemu-4-build.sh
+++ b/.github/workflows/scripts/qemu-4-build.sh
@@ -47,6 +47,7 @@ function freebsd() {
 }
 
 function linux() {
+  EXTRA_CONFIG="${1:-}"
   echo "##[group]Autogen.sh"
   run ./autogen.sh
   echo "##[endgroup]"
@@ -56,7 +57,7 @@ function linux() {
     --prefix=/usr \
     --enable-pyzfs \
     --enable-debug \
-    --enable-debuginfo
+    --enable-debuginfo $EXTRA_CONFIG
   echo "##[endgroup]"
 
   echo "##[group]Build"
@@ -139,6 +140,9 @@ case "$1" in
   debian*|ubuntu*)
     deb_build_and_install
     ;;
+  arch*)
+    linux "--sysconfdir=/etc --enable-linux-experimental"
+    ;;
   *)
     linux
     ;;
```